### PR TITLE
Private Channels view

### LINKED ISSUE #456: Private channels tab instead of Favorites
The second tab should show Private channels instead of Favorites.

https://zpl.io/V19YOrm

All Private channels should show the contacts

https://zpl.io/b6gY49j

Private channels should show only the posts from you and the other participant

https://zpl.io/25A3Jd8

On the Info page you can delete the contact

https://zpl.io/boLr8Ov

### PR DESCRIPTION
**Pull request by agazso**
Fixes #456 .

### CODE REVIEW

**Comment by agazso** [comment]
This PR adds the `Private channels` tab, the `All private channels` and `Private channel page` screens from the current design.

All references to Favorites are removed.

File: src/actions/asyncActions.ts [hunk 1]
```diff
@@ -119,6 +126,58 @@ export const AsyncActions = {
...
+                                uri: contactFeed.feedUrl,
+                                image: contact.image,
+                            },
+                        };
+                        allPosts.push(postWithAuthor);
+                    }
+                }
+            }
+            const posts = mergeUpdatedPosts(allPosts, previousPosts);
+            dispatch(Actions.updateRssPosts(posts));
```
Comment (vujevits) [inline src/actions/asyncActions.ts]: This action should be renamed

File: src/actions/asyncActions.ts [hunk 2]
```diff
@@ -119,6 +126,58 @@ export const AsyncActions = {
             dispatch(Actions.updateRssPosts(posts));
         };
     },
+    downloadPrivatePostsFromContacts: (contactFeeds: ContactFeed[]): Thunk => {
```
Comment (vujevits) [inline src/actions/asyncActions.ts]: Is there a way to refresh when you are on a view with a specific 1to1 feed?

Comment (agazso) [reply src/actions/asyncActions.ts]: You can pass in an array with one element.

File: src/actions/asyncActions.ts (hunk 1 above)
Comment (agazso) [reply src/actions/asyncActions.ts]: This updates the `rssPosts` field in the AppState, hence the name.

What is your suggestion?

**Pull request merged**